tools/tests/s3tests-runner.sh

```diff
@@ -187,22 +188,33 @@ _run() {
 
   echo "$test : $result"
 
-  ( flock -w 5 255 ; yq -i \
-    ".tests += [{\"name\": \"${name}\", \"result\": \"${result}\", \"slot\": \"${slot}\", \"time\": \"$runtime\"}]" \
-    "${TMPFILE}" ; ) 255>"${TMPFILE}.lock"
+  if [ "$CONTAINER_CMD" = "docker" ] ; then
+    docker logs "$CONTAINER" > "${OUTPUT_DIR}/logs/${test}/radosgw.log" 2>&1
+  elif [ -n "$CONTAINER" ] ; then
+    mv "${OUTPUT_DIR}/logs/radosgw.log" "${OUTPUT_DIR}/logs/${test}/radosgw.log"
+  fi
+
+
+  ( flock -w 5 255 ; \
```

Comment:
Why share TMPFILE between processes? Could this write into separate files and have a combine step shortly before _convert? 
 
TMPFILE could use a better name that includes what is in there.

Reply:
Done. Collect results in separate files, merge files in the `_convert` step